frontend/jupyter/src/app/pages/form/form-new/form-image/form-image.component.html

```diff
@@ -56,8 +56,8 @@
         "
       >
         <mat-icon class="server-type-icon" svgIcon="group-two"></mat-icon>
-        <p class="server-type-title">Remote Desktop</p>
-        <p class="server-type-content">A ubuntu remote desktop.</p>
+        <p class="server-type-title">Ubuntu</p>
+        <p class="server-type-content" i18n>A ubuntu remote desktop.</p>
```

Comment:
An Ubuntu remote desktop?

Reply:
That does seem better with english rules in mind of "an" before vowels. Good catch